emmaa/model_tests.py

```diff
@@ -72,6 +74,26 @@ def run_tests(self):
         for (stmt, result) in results:
             self.add_result(result)
 
+    def make_english_result(self, result):
+        """Create an English description of a path."""
+        stmts = stmts_from_path(result.paths[0], self.pysb_model,
```

Comment:
Probably more flexible for downstream use if each sentence corresponding to each statement is stored as an entry in a list.